### PR TITLE
Add Amplitude to Rococo

### PR DESCRIPTION
**Pull request by gonzamontiel**
I don't know why, there are changes that are already merged from a last PR, but that are still shown here. Any clue why the diff is also showing old changes? 

The intended changes are: 
- Add Amplitude to Rococo (ac297a8fd9df6c95a643cd782d21dd899bcaa4e0)

The rest should already be merged due to https://github.com/polkadot-js/apps/pull/7807.

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 1]
```diff
@@ -25,6 +25,14 @@ export const prodParasKusama: EndpointOption[] = [
...
     }
   },
+  {
+    info: 'amplitude',
+    homepage: 'https://pendulumchain.org/amplitude',
+    paraId: 2124,
+    text: 'Amplitude',
+    isUnreachable: true,
+    providers: {} // Working on making this live ASAP
+  },
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:35]: This does look like it is there -

https://github.com/polkadot-js/apps/blob/a6cc58e582ea60c3f220e23b36d563220b1ca136/packages/apps-config/src/endpoints/productionRelayKusama.ts#L28-L35

File: packages/apps-config/src/endpoints/testingRelayRococo.ts [hunk 2]
```diff
@@ -23,6 +23,14 @@ export const testParasRococo: EndpointOption[] = [
...
     }
   },
+  {
+    info: 'rococoAmplitude',
+    paraId: 2124,
+    text: 'Amplitude',
+    providers: {
+      PendulumChain: 'wss://pencol-roc-00.pendulumchain.tech'
+    }
+  },
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRelayRococo.ts:33]: This is indeed missing (also clueless as to why - all I can think of is a conflict and unintentional drop along the way) -

https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/endpoints/testingRelayRococo.ts

Comment (gonzamontiel) [reply packages/apps-config/src/endpoints/testingRelayRococo.ts:33]: Hey Jaco, I think our forked master diverged from the upstream, and from that point it is broken. I created another PR that should be the valid one. Will close this with a comment below linking it.

**Comment by gonzamontiel** [comment]
Closing this in favor of https://github.com/polkadot-js/apps/pull/7950

**Pull request closed without merging**